tiledb/sm/query/reader.h

```diff
@@ -934,6 +940,9 @@ class Reader {
    */
   bool offsets_extra_element_;
 
+  /** The offset bitsize used for variable-sized attributes. */
+  int offsets_bitsize_;
```

Comment:
- Member variables of primitive types are not default initialized. We should initialize this to `constants::cell_var_offset_size * 8`.
- Also make this a `uin32_t`.